build-configs/aschaffenburg/assets/app-logo-alone.svg

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What does the `alone` mean in the file name?

Reply:
I think we should use the same naming for this and for the inverted integreat G. I don't mind which one though.